app/pods/cases/case/subcases/subcase/overview/controller.js

```diff
@@ -16,6 +17,10 @@ export default class CasesCaseSubcasesSubcaseOverviewController extends Controll
   @tracked governmentAreas;
   @tracked siblingSubcasesCount;
 
+  get showMandateesNotApplicableMessage() {
+    return [CONSTANTS.SUBCASE_TYPES.BEKRACHTIGING].includes(this.model.subcase.get('type')?.get('uri'));
```

Comment:
Is the `get()` for `'type'` required?  `this.model.subcase` is not a proxy object, isn't it?

Reply:
You're right, not needed here. The semantics when it comes to proxy objects still confuse me a bit :woozy_face: